documentation/migration.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Migration
+template: default.ejs
+theme: light
+repository: https://github.com/unexpectedjs/unexpected
+---
+
+# Migration
+
+Unexpected has from its inception taken backwards compatibility very
+seriously. We think this is particularly important for a library
+intended for use in testing - we recognise that comprehensive test
```

Comment:
```suggestion
intended for use in testing &ndash; we recognise that comprehensive test
```

(seems like our markdown processor does not support `--` as a shorthand for an "n dash", I thought that was standard)